### DISCUSSION TITLE
Github Projects notification

### DISCUSSION THREAD

**Discussion by xii-hcPark** [community/community#40610] [Projects and Issues]
### Select Topic Area

Question

### Body

Hi I am currently using github projects feature.
There is an inconvenience, but even if you create a task in projects, notifications do not come.
It would be nice to create a function that allows you to receive notifications via Slack or email.

**Comment by ossanna16** [comment]
Hi there @xii-hcPark and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Reply by Julian-Dumitrascu** [reply]
What were those chats?
What's their status?

**Comment by MrCatEsq** [comment]
It would be nice to get an email notification every time if something in project changed: a card/task created, modified, moved, deleted... It would be extremely nice if we will see who, when, what and where changed in the notification email.

**Comment by rogue-shadowdancer** [comment]
and I think there is a filter or panel in the GitHub Project to watch the recently changed tasks may also be a good choice.

**Reply by rogue-shadowdancer** [reply]
https://github.com/adgcz/project-activity
may look like this action, and I think if this filter can be integrated in the GitHub website would be great. but I think the action of repo projects can not be used in the organization's projects(since the project in an organization may have many repos).

**Reply by davetapley** [reply]
@rogue-shadowdancer see also: https://github.com/orgs/community/discussions/180056

**Comment by nbarsova** [comment]
Bumping this up. As it is  not possible to get email notifications for tasks that are not connected to a repository artifact, it very hard to use github projects in cross-functional teams.

**Comment by Uaitt** [comment]
Does anyone know if this feature is being worked on? It would be extremely helpful in my opinion 🙏 😇

**Comment by sigubrat** [comment]
This is really important. It's so hard to stay aware of progress with the current situation

**Comment by alper** [comment]
Just like you can receive notifications for everything happening in a repository, it should be possible to have the same for a project.

**Comment by KMaisel** [comment]
Bumping this, we would love to have a feature like this! We use projects to plan app releases and would love to get notified when something is added to a project so that we can ensure QA is done prior to release.

**Comment by sam-artuso** [comment]
Hard to believe that there's no way to set up notifications!

**Comment by davetapley** [comment]
A use case I naively assumed there would be a notification for:
When I change an issue's status to 'In review' I'd like the user who opened the issue to be notified, so they can review it...

**Comment by Alloc86** [comment]
Yeah, missing any kind of notifications basically makes these Projects unusable for me. Can't go in every day and manually check for any kind of modifications ...
